### PR TITLE
zwx: article drag add

### LINKED ISSUE #142: 路书编辑器：拖拽添加文献节点

### PR DESCRIPTION
**Pull request by zwx980624**
article drag add
close #142

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "github-markdown-css": "^4.0.0",
     "global": "^4.4.0",
     "json-loader": "^0.5.7",
+    "less-loader": "4.1.0",
```
Comment (luo-junyu) [inline package.json:22]: please notify us to yarn install 😂

File: src/views/RoadmapEditorView.vue [hunk 2]
```diff
@@ -798,4 +825,7 @@ export default {
     margin-left: 40px;
     margin-right: 40px;
   }
+  .ghost {
```
Comment (luo-junyu) [inline src/views/RoadmapEditorView.vue:828]: What is ghost??

**Review by luo-junyu — APPROVED** [review]
OK! 
pass
🤖

File: package.json (hunk 1 above)
Comment (zwx980624) [reply package.json:22]: okok, please yarn install before run dev

File: src/views/RoadmapEditorView.vue (hunk 2 above)
Comment (zwx980624) [reply src/views/RoadmapEditorView.vue:828]: Ghost is used in the vue-dragged wheel.

File: package.json (hunk 1 above)
Comment (luo-junyu) [reply package.json:22]: 👍

**Pull request merged**